x/liquidity/spec/01_concepts.md

```diff
@@ -1,60 +1,83 @@
-<!--
-order: 1
--->
+<!-- order: 1 -->
 
-# Concepts
+ # Concepts
 
-## The Liquidity module on the Cosmos-SDK
+## Liquidity Module
 
-The liquidity module serves AMM style decentralized exchange on the Cosmos-SDK. AMM style exchange provides unique coin swap model for its users, liquidity providers and swap requestors.
+The Liquidity Module lets you implement a decentralized token exchange (DEX) on any Cosmos SDK-based network. Any user can create a liquidity pool with a pair of tokens, provide liquidity by depositing reserve tokens into the liquidity pool, and trade tokens using the liquidity pool.
 
-### Democratized Liquidity Providing
+### How the Liquidity Module works
 
-AMM allows liquidity providers to play market maker roles without technically sophisticated real-time orderbook management and significant capital requirement. The liquidity provides only need to deposit coins into liquidity pools, and monitor asset composition changes and accumulated fee rewards from liquidity providing.
+These features of the Liquidity Module create incentives to transfer tokens:
 
-It results in democratized liquidity providing activities, hence lowering the cost of liquidity and more enriched quality liquidity provided on the AMM exchange.
+- Combines a traditional order book-based exchange system with a Uniswap-like AMM (Automated Market Maker) mechanism. This hybrid system deepens liquidity for the token swap marketplace.
+
+- Executes batch-style swaps that minimize front-running risk and sub-second latency competition, thereby protecting ordinary traders.
+
+  - The order book accumulates incoming limit orders into a batch.
+  - The Liquidity Module matches accumulated limit orders and orders from the liquidity pool at an equivalent swap price at each batch execution height.
+  - All limit orders in a batch are treated equally and executed at the same swap price.
+
+### Democratized Liquidity
+
+Democratized liquidity lowers the cost of liquidity and provides an enriched quality liquidity provided on the AMM exchange.
+
+AMM allows liquidity providers to play market maker roles without investing in technically sophisticated real-time orderbook management software or making significant capital investments.
 
 ### Liquidity Pool
 
-Liquidity pool is a coin reserve with two kinds of coins to provide liquidity for coin swap requests between the two coins in the liquidity pool. The liquidity pool acts as the opposite party of swap requests as the role of market makers in the AMM style exchange.
+A liquidity pool contains two assets in a trading pair. A liquidity pool is a coin reserve with a pair of tokens to provide liquidity for coin swap requests between the two coins. The liquidity pool acts as the opposite party of swap requests as the role of market makers in the AMM style exchange.
```

Comment:
@dongsam help me understand when we use "token" and when we use "coin" and when we use "asset". Are these terms talking about the same thing?

Reply:
is our liquidity pool like the Binance liquidity pool?  https://academy.binance.com/en/articles/what-are-liquidity-pools-in-defi 
if there are differences, what are they? (I'm looking to ground my understanding, I have so much to learn)

Reply:
> @dongsam help me understand when we use "token" and when we use "coin" and when we use "asset". Are these terms talking about the same thing?

We probably need to consistently use "coin" for the document. Asset has some different nuance when we want to emphasize the value of the coins.